mk/spksrc.compile.mk

```diff
@@ -33,7 +33,7 @@ compile_msg:
 pre_compile_target: compile_msg
 
 compile_target:  $(PRE_COMPILE_TARGET)
-	@$(RUN) $(MAKE) $(COMPILE_MAKE_OPTIONS)
+	@$(RUN) $(MAKE) -j$(NCPUS) $(COMPILE_MAKE_OPTIONS)
```

Comment:
i'm not sure if this does anything

Reply:
yes it is obsolete, as `COMPILE_MAKE_OPTIONS += -j$(NCPUS)` is defined in spksrc.common.mk

Reply:
Thanks